deploy/crd.yaml

```diff
@@ -32,6 +32,9 @@ spec:
     - name: v1-5-0
       storage: false
       served: true
+    - name: v1-6-0
```

Comment:
Is it possible to keep the versions for all resource types in sync?

Reply:
> Is it possible to keep the versions for all resource types in sync?

@benlangfeld Could you please be more specific. I didn't catch your question.

Reply:
When you release new versions, you only bump the version of the PXC type, but not the other types defined here, like backups. This causes problems with some of our tooling, so we manually include new versions of the other types in a fork of this definition. I'm asking if you could possibly ensure that all defined types provide the same versions on every release so as not to break third-party deployment tooling.

Reply:
@benlangfeld Is your tooling using the latest master branch or you checkout the release tag (`git clone -b v1.5.0 https://github.com/percona/percona-xtradb-cluster-operator`)? Release tag should give you consistent versions and if not then it's probably a bug.
This commit here is not related to the release - it is something done after the release to prepare running tests on the master branch (although it can be questioned also if it can be done differently).

Reply:
We grab the CRD from a release. I think you misunderstood. The problem is this:

![image](https://user-images.githubusercontent.com/210221/89034147-c49f5280-d30e-11ea-9439-aff94c8337f8.png)

![image](https://user-images.githubusercontent.com/210221/89034160-cbc66080-d30e-11ea-8c72-a5a4668d6576.jpeg)

The list of versions for all resource types except the cluster itself are incomplete. The same versions should be served for all resource types defined by the operator.